framework/ChiPhysics/PhysicsMaterial/MultiGroupXS/multigroup_xs.h

```diff
@@ -0,0 +1,146 @@
+#ifndef CHI_PHYSICS_TRANSPORT_CROSS_SECTIONS_H
+#define CHI_PHYSICS_TRANSPORT_CROSS_SECTIONS_H
+
+#include "multigroup_xs_base.h"
+
+
+/**\defgroup LuaTransportXSs Transport Cross Sections
+ * \ingroup LuaPhysics*/
+
+namespace chi_physics
+{
+
+
+//######################################################################
+/**
+ * A class for handling forward multi-group cross sections.
```

Comment:
Remove the word "forward" please. This should just be classical multi-group cross sections.